notebooks/official/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -32,18 +32,18 @@
         "<table align=\"left\">\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def bq_query(sql):
</pre><p>Add a type-hint of <code>sql: str</code></p><p> </p><p>Also add return type.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
adding sql: str as type hint for input. 

However, not all BigQuery queries result in the same type. For SELECT queries, it returns a Pandas DataFrame. For other queries (e.g. CREATE SCHEMA to create a BQ dataset, or to run a BQML job) the function returns a str. So changing the def line to:

`def bq_query(sql: str) -> Union[str, pd.DataFrame]`